src/sentry/shared_integrations/client/base.py

```diff
@@ -481,9 +499,16 @@ def disable_integration(self, buffer) -> None:
         )
 
         if (
-            features.has("organizations:slack-fatal-disable-on-broken", org)
-            and rpc_integration.provider == "slack"
-        ) and buffer.is_integration_fatal_broken():
+            (
+                features.has("organizations:slack-fatal-disable-on-broken", org)
+                and rpc_integration.provider == "slack"
+            )
+            and buffer.is_integration_fatal_broken()
+        ) or (
+            features.has("organizations:github-disable-on-broken", org)
+            and rpc_integration.provider == "github"
+        ):
```

Comment:
should we be doing a similar check for `is_integration_fatal_broken` if it is github or is that smth we only want to do for slack?

Reply:
^^ it doesn't seem like we're checking if it's broken before disabling right now.

Reply:
This is within the disable function which is called after `buffer.is_integration_broken()` is `True`. We did `is_integration_fatal_broken` for slack since we wanted to turn on the slack flag but only validated the fatal broken integrations. I'm not sure if we will have the same situation with github but we can see what we get in the logging and then decide?